website/docs/docs/build/data-tests.md

```diff
@@ -251,6 +261,29 @@ Note that, if you select to store test failures:
 * Test result tables are created in a schema suffixed or named `dbt_test__audit`, by default. It is possible to change this value by setting a `schema` config. (For more details on schema naming, see [using custom schemas](/docs/build/custom-schemas).)
 - A test's results will always **replace** previous failures for the same test.
 
+<VersionBlock firstVersion="1.8">
+
+## New `tests:` syntax
+
+Data tests were historically called "tests" in dbt as the only form of testing available. With the introduction of unit tests in v1.8, it was necessary to update our naming conventions and syntax. As of v1.8,  `tests:` is still supported in your YML configuration file as an alias but will be deprecated in the future in favor of `data_tests:`. 
+
+As we progress towards this deprecation, the examples in our docs pages will be updated to reflect this new syntax, but we highly recommend you begin the migration process as soon as you upgrade to v1.8 to avoid interruptions or issues in the future.
+
```

Comment:
do you think it makes sense to move this section/content into the Upgrade to v1.8 guide? it can help make this page stay more evergreen sans version blocks

for "before and after" examples, we do have some prior art here (look at the second example) https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-schema-environment#example-queries

Reply:
It will be there too but we need to call this out in the doc before we change all of the examples. Users may not look at that guide. I'll make this the last version as well to help

Reply:
the content reads very much like "release note" material (snapshot in time, not evergreen)

or there plans to remove this section after some date? the syntax will eventually not be "new" anymore

Reply:
Yes, I version blocked it so it only shows in 1.8. Everything in the upgrade guide is elsewhere in the docs as well.